feat: support --package to specify the package to profile (#61)
cmyr/cargo-instruments: README.md

$ open target/instruments/mybin_Allocations_2021-05-09T12:34:56.trace
```

If there are mutliple packages, you can specify the package to profile with
the `--package` flag.

For example, you use Cargo's workspace to manage multiple packages. To profile
the bin `bar` of the package `foo`:

```sh
$ cargo instruments --package foo --template alloc --bin bar
```

> In most of time, the package only has one binary which makes `--package` behaves the

Reviewer: ```suggestion
In many cases, a package only has one binary. In this case `--package` behaves the
```

(Is this accurate?)

Author: Yes. It's more precise.

            --bin <NAME>                   Binary to run
            --example <NAME>               Example binary to run
            --features <CARGO-FEATURES>    Features to pass to cargo

>             --package <NAME>               Specify package for example/bin/bench

Reviewer: ```suggestion
            --package <NAME>               Specify package to search for target (in a cargo workspace)
```
